src/components/InterviewScheduleModal.tsx

```
@@ -0,0 +1,411 @@
+          timezone: Intl.DateTimeFormat().resolvedOptions().timeZone,
+          topics: [],
+          additionalNotes: '',
+        })
+        setIsSubmitted(false)
+        setError(null)
+      }, 300)
+    }
+  }
+
+  return (
+    <div className='modal-overlay' onClick={handleClose}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)